includes/application.php

```diff
@@ -46,7 +46,7 @@ function ($class_name) {
 
     // Validate session token, if exists
     if (!empty($_SESSION['session_token'])) {
-        $UserAuth = FirstAide\Authentication::withSessionToken($DB, $_SESSION['session_token']);
+        $UserAuth = FirstAide\Authentication::createInstanceWithSessionToken($DB, $_SESSION['session_token']);
```

Comment:
I have already tried doing this but seems like we can not get rid of using namespaces and importing it directly. https://stackoverflow.com/questions/9317022/troubleshooting-the-use-statement-with-non-compound-name-has-no-effect
<img width="991" alt="screen shot 2017-07-24 at 23 03 53" src="https://user-images.githubusercontent.com/13470643/28536218-9c464bea-70c4-11e7-88e9-7d59afc2e122.png">